llvm/test/CodeGen/ARM/scmp.ll

```diff
@@ -0,0 +1,155 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=thumbv7-apple-darwin %s -o - | FileCheck %s
```

Comment:
This should use `armv7` rather than `thumbv7` (though you can also keep the thumb coverage by moving it into the Thumb/ directory).